features/car/bridge.feature

```diff
@@ -48,8 +48,8 @@ Feature: Car - Handle driving
             | from | to | route           | modes                           | speed   | time     |
             | a    | g  | abc,cde,efg,efg | driving,driving,driving,driving | 13 km/h | 340s +-1 |
             | b    | f  | abc,cde,efg,efg | driving,driving,driving,driving | 9 km/h  | 318s +-1 |
-            | c    | e  | cde,cde         | driving,driving                 | 5 km/h  | 295s +-1 |
-            | e    | c  | cde,cde         | driving,driving                 | 5 km/h  | 295s +-1 |
+            | 1    | 2  | abc,cde,efg     | driving,driving,driving         | 7 km/h  | 306s +-1 |
```

Comment:
Would it be better if tests specify also source and  target bearings to avoid changes in expected values?